watchdog-config/extra-sql-queries.yml

```diff
@@ -75,3 +75,400 @@ users_with_insificient_disk:
+  query: |
+    SELECT
+        feedback.partner_name,
+        reported::varchar(255),
+        detail,
+        count(*) AS detail_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    WHERE source = 'manual'
+    GROUP BY
+        feedback.partner_name,
+        reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - detail_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+automatic_feedback_docs:
+  query: |
+    SELECT
+        feedback.partner_name,
+        reported::varchar(255),
+        detail,
+        count(*) AS detail_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_feedback_docs AS feedback
+    LEFT JOIN
+        monitoring_urls ON feedback.partner_name=monitoring_urls.partner_name
+    WHERE source = 'automatic'
+    GROUP BY
+        feedback.partner_name,
+        reported,
+        source,
+        detail,
+        cht_instance
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - reported:
+        usage: "LABEL"
+    - detail:
+        usage: "LABEL"
+    - detail_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+replication_by_status:
+  query: |
+    SELECT
+        status.partner_name,
+        period_start::varchar(255),
+        replication_success_count,
+        replication_failure_count,
+        replication_denied_count,
+        replication_error_count,
+        replace(monitoring_urls.url, 'https://', '') as cht_instance
+    FROM app_monitoring_replication_by_status AS status
+    LEFT JOIN
+        monitoring_urls ON status.partner_name=monitoring_urls.partner_name
+  metrics:
+    - partner_name:
+        usage: "LABEL"
+    - period_start:
+        usage: "LABEL"
+    - replication_success_count:
+        usage: "GAUGE"
+    - replication_failure_count:
+        usage: "GAUGE"
+    - replication_denied_count:
+        usage: "GAUGE"
+    - replication_error_count:
+        usage: "GAUGE"
+    - cht_instance:
+        usage: "LABEL"
+
+# replication_activity:
+#   query: |
+#     SELECT
+#         sms.partner_name,
+#         reported::varchar(255),
+#         delta_of_doc AS delta_docs,
+#         replace(monitoring_urls.url, 'https://', '') as cht_instance
+#     FROM app_monitoring_sms_error_and_users_docs_replication AS sms
+#     LEFT JOIN
+#         monitoring_urls ON sms.partner_name=monitoring_urls.partner_name
+#   metrics:
+#     - partner_name:
+#         usage: "LABEL"
+#     - reported:
+#         usage: "LABEL"
+#     - delta_docs:
+#         usage: "GAUGE"
+#     - cht_instance:
+#         usage: "LABEL"
+
+initial_replications:
+  query: |
+    SELECT
```

Comment:
is the plan to chart these over time?  I see, for example `bhojpur-ne`  returns 61 results and only 55 with   `count_initial_replications` values of  not `0` 

just making sure this is helpful data to query!

Reply:
Great suggestion limited to results with `count_initial_replications > 0`

Reply:
great! I see just about every query has this now - nice work.